### ISSUE TITLE
Object.activate is deprecated.

### ISSUE THREAD

**Issue by Arcanemagus** [AtomLinter/linter-phpmd#8]
AtomLinter v0.X.Y API has been deprecated. Please refer to the Linter docs to update and the latest API: https://github.com/AtomLinter/Linter/wiki/Migrating-to-the-new-API
```
Object.activate (C:\Users\Landon\.atom\packages\linter\lib\main.coffee:57:8)
Package.activateNow (C:\Users\Landon\AppData\Local\atom\app-1.0.0\resources\app.asar\src\package.js:240:19)
<unknown> (C:\Users\Landon\AppData\Local\atom\app-1.0.0\resources\app.asar\src\package.js:221:30)
Package.measure (C:\Users\Landon\AppData\Local\atom\app-1.0.0\resources\app.asar\src\package.js:165:15)
Package.activate (C:\Users\Landon\AppData\Local\atom\app-1.0.0\resources\app.asar\src\package.js:213:14)
PackageManager.activatePackage (C:\Users\Landon\AppData\Local\atom\app-1.0.0\resources\app.asar\src\package-manager.js:452:21)
```

**Comment by Arcanemagus** [comment]
Looks like this is the windfall of https://github.com/AtomLinter/linter-phpmd/issues/7 not having been dealt with.

**Comment by MagneticMule** [comment]
+1

**Comment by fredcamps** [comment]
+1

**Comment by Mihai-P** [comment]
+1

**Comment by Matamanthemaster** [comment]
:+1:

**Comment by rrd108** [comment]
+1

**Comment by steelbrain** [comment]
Fixed by #11
